app/assets/stylesheets/navbar/components/_navbar_mobile.scss

```diff
@@ -0,0 +1,102 @@
+@mixin style1 {
+  margin-left: auto;
+  margin-right: auto;
+  margin-bottom: 5px;
+  width: 100px;
+}
+
+@mixin style2 {
+  border:2px solid white;
+  border-radius: 5px;
+}
+
+.navbar {
+  background-color: #08355e;
```

Comment:
I suggest you save that color in a variable so that you can easily use it in other properties